irobotclient/main.py

```diff
@@ -0,0 +1,88 @@
+"""
+Copyright (c) 2017 Genome Research Ltd.
+
+This program is free software: you can redistribute it and/or modify it
+under the terms of the GNU General Public License as published by the
+Free Software Foundation, either version 3 of the License, or (at your
+option) any later version.
+
+This program is distributed in the hope that it will be useful, but
+WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
+Public License for more details.
+
+You should have received a copy of the GNU General Public License along
+with this program. If not, see <http://www.gnu.org/licenses/>.
+"""
+
+import argparse
+import string
+import os
+
+from irobotclient.client import IrobotClient
+
+
+def _get_command_line_agrs():
+    """
+    Get program arguments
+
+    :return: args
+    """
+
+    parser = argparse.ArgumentParser(prog="irobot-client",
+                                     formatter_class=argparse.RawTextHelpFormatter,
+                                     description="Command line interface for iRobot HTTP requests",
+                                     usage="irobot-client [options] INPUT_FILE OUTPUT_FILE")
+    parser.add_argument("input_file", help="path and name of input file")
+    parser.add_argument("output_file", help="path and name of output file")
+    parser.add_argument("-u", "--url", help="URL scheme, domain and port for irobot. EXAMPLE: http://irobot:5000/")
```

Comment:
`http://irobot:5000` is a valid URL. Your code should canonicalise all input to have a trailing slash (or not, just as long as it's consistent)